### ISSUE TITLE
Editor: Feature Missing - Unable to see already linked DRDs for a component

### ISSUE THREAD

**Issue by lukas-foltyn-profinit** [apache/incubator-kie-tools#1355]
**Problem:**

Imagine that I have a DRG with several decision and I decided to "hide" the decision complexity using DRDs.
So, I would design the DRG first, and then I would one by one add DRD for each DRG decision.

There is a quick and simple way, how to add a decision from DRG to a DRD - using DRD Actions ("share" icon on the component), Create.

The problem is that on the DRG level you are not able to see to which DRD(s) is the DRG decision linked to. The only way how to "drill down" from DRG to the linked DRD is to go and search for the DRG decision name in Decision Navigator / Decision Graphs. In case you have tens or hundreds of DRDs, this becomes extremely user unfriendly (in the VS Code plugin, you cannot even use Ctrl+F to search in DRD list and you have to scroll through all Decision Graphs)

The situation is the same, if you use DRD decision in another DRD diagram.

**Suggestion:**

Add a list of all linked DRDs on the component in the editor.
If the user clicks on one of the DRDs in the list, it gets the user to the selected DRD.

Such list could be added as a new icon for the component in the editor or as part of the DRD Actions list.
![image](https://user-images.githubusercontent.com/106966707/205261001-e31f3f9b-3f72-4dcf-84f9-d4af9f293bb4.png)

Thank you in advance for looking into this issue.

Regards,
Lukas

**Comment by yesamer** [comment]
@tiagobento I guess the new editor improved the DRDs management, that's right?

**Comment by lukas-foltyn-profinit** [comment]
I have checked the latest version of DMN Editor - v0.32.0 from 2020-11-23 and it seems the same.

At the moment, I am missing a way, how to search in DRDs. The search field works only with the opened screen on the left hand side, but not with the tree on the right hand side - that would be the bare minimum.

![image](https://github.com/kiegroup/kie-tools/assets/106966707/1b9d2d21-1dc7-462d-9ddd-b688e029f509)

**Comment by yesamer** [comment]
@lukas-foltyn-profinit Correct, the current 0.x stream doesn't have any change in that area. In my previous comment, I was referring to the new DMN Editor we are currently implementing (https://github.com/kiegroup/kie-tools/pull/2064)

**Comment by tiagobento** [comment]
Can you please try the new DMN Editor at https://sandbox.kie.org/ after our Apache KIE 10.0.0 release?

Hiding a node from a DRD can be done by pressing the `x` key.
Pressing `Backspace` will remove the selected nodes from the DRG and from all DRDs.

**Comment by tiagobento** [comment]
Closing but still interested in hearing back from @lukas-foltyn-profinit 😊
